src/components/POLineCard/POLineCard.test.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import '@folio/stripes-erm-components/test/jest/__mock__';
+import { StaticRouter as Router } from 'react-router-dom';
+import { renderWithIntl } from '@folio/stripes-erm-components/test/jest/helpers';
+import POLineCard from './POLineCard';
+
+
+const poLine = {
+  'acquisitionMethod': 'Purchase',
+  'id': '556abc25-ebbf-3fb2-b478-1bfaff0af4dc',
+  'poLineNumber': '81-1',
+  'titleOrPackage': 'ABA Journal',
+};
+
+const poLineWithInstanceId = { ...poLine };
+poLineWithInstanceId.instanceId = 'abc';
+
+const data = {
+  'headerEnd': '<WithStripes(Pluggable) />',
+  'id': 'edit-poline-0-0',
+  'poLine': {
+    'id': '556abc25-ebbf-3fb2-b478-1bfaff0af4dc',
+    'edition': 'First edition',
+    'checkinItems': false,
+    'agreementId': '09c6ed1b-3984-4d9a-8f9b-e1200b68b61c',
+    'acquisitionMethod': 'Purchase',
+    'alerts': '[]',
+    'cancellationRestriction': false,
+    'cancellationRestrictionNote': '',
+    'claims': '[{…}]',
+    'collection': false,
+    'contributors': '[]',
+    'cost': '{additionalCost: 0, currency: "USD", discount: 0, d…}',
+    'description': '',
+    'details': '{productIds: Array(1), receivingNote: "", subscript…}',
+    'donor': '',
+    'fundDistribution': '[{…}]',
+    'isPackage': false,
+    'locations': '[{…}]',
+    'orderFormat': 'Other',
+    'paymentStatus': 'Pending',
+    'physical': '{materialSupplier: "e0fb5df2-cdf1-11e8-a8d5-f2801f1…}',
+    'poLineDescription': '',
+    'poLineNumber': '81-1',
+    'publicationDate': '1915',
+    'publisher': 'American Bar Association',
+    'purchaseOrderId': 'c27e60f9-6361-44c1-976e-0c4821a33a7d',
+    'receiptStatus': 'Pending',
+    'reportingCodes': '[]',
+    'requester': '',
+    'rush': false,
+    'selector': '',
+    'source': 'User',
+    'tags': '{tagList: Array(1)}',
+    'titleOrPackage': 'ABA Journal',
+    'vendorDetail': '{instructions: "", noteFromVendor: "", referenceNum…}',
+    'metadata': '{createdDate: "2021-08-04T01:50:10.921+00:00", upda…}',
+  },
+  'children': null,
+};
```

Comment:
Can you replace this data object with fresh data by recreating a POLine Card on the UI and using the devtools to get the props passed into this component. For eg: `headerEnd` is not a valid prop being passed to the `POLineCard` component.